client/my-sites/plans/plan-overview/plan-status/index.jsx

```diff
@@ -55,12 +81,14 @@ const PlanStatus = React.createClass( {
 								} )
 							}
 						</h1>
+						{ this.renderNotice() }
 					</div>
 
 					<Button
 						primary={ getDaysUntilUserFacingExpiry( this.props.plan ) < 6 }
 						className="plan-status__button"
-						onClick={ this.purchasePlan }>
+						onClick={ this.purchasePlan }
+					>
```

Comment:
Did you return to the line on purpose? In that case we should probably discuss it internally to make sure everybody is on the same page as I don't remember seeing this approach very often in our code base.